### DISCUSSION TITLE
Migration of UI kit from Sketch to Figma?

### DISCUSSION THREAD

**Discussion by lordsfantasy** [carbon-design-system/carbon#9872] [Help] [answered]
Hi Carbon team, any plans on this?

**Comment by joshblack** [comment]
Hi there @lordsfantasy! 👋 

I believe there is work happening on adding support for Figma. @aagonzales or @laurenmrice can add more info 👀

**Comment by aagonzales** [comment, marked as answer]
Hello, yes we are in the process of building a figma kit out right now. The plans are to release the white theme at the end of Q4 and the other themes early in 2022.
